tests/acceptance/test_organization_user_feedback.py

```diff
@@ -27,13 +29,16 @@ def setUp(self):
 
     def test(self):
         with self.feature('organizations:sentry10'):
-            self.create_group(
-                project=self.project,
-                message='Foo bar',
-            )
-            self.create_userreport(group=self.group, project=self.project, event=self.event)
+            self.create_userreport(
+                date_added=datetime(
+                    2013,
+                    5,
+                    7),
```

Comment:
You could use `timezone.now()` it has a fixed time in acceptance tests.

Reply:
Ah right, previously when I tried that, I hadn't changed the factory `create_userreport` to use `date_added` and `timezone.now()` doesn't seem to be mocked in the model (where `date_added` by default is `timezone.now`)